docs/GETTING_STARTED.md

```diff
@@ -174,24 +174,27 @@ The server needs access to some of BW's data files in order to generate map imag
 editor, such as [this one](http://www.zezula.net/en/mpq/download.html) and make sure to download
 "listfiles" from that website as well, which you'll need to use in the mpq editor. Use the mpq
 editor to extract BW's data files from `stardat.mpq`, `broodat.mpq`, in that order, having
-`broodat.mpq` overwrite any conflicting files from `stardat.mpq`. The necessary directories in
+`broodat.mpq` overwrite any conflicting files from `stardat.mpq`. 
+You can obtain `stardat.mpq` and `broodat.mpq` directly from Blizzard by downloading the final release of
+[StarEdit](http://download.blizzard.com/pub/starcraft/StarEdit/StarEdit.zip). You can also obtain those files from
+an old Starcraft Broodwar version (such as 1.16) installation folder. Note that you cannot find those files from SC:Remastered.
```

Comment:
_"Note that you cannot find those files from SC:Remastered."_ - This is not technically true ^^
Those files are still available in SC:R, but a bit harder to get. They now need to be extracted from a CASC container file using some program that can read those, e.g. [CascView](http://www.zezula.net/en/casc/main.html).

Honestly, I would just leave that line out, and perhaps even the one about using 1.16 installation folder. The method of using the StarEdit seems to be version-agnostic, so I would just mention that method to make things simpler.

Also, there might be some importance of casing for the name of those files that we might want to mention, but I'm not sure atm, so I'll leave someone else to comment on that ^^.

Reply:
Thanks for the comment I will remove the line I agree with you :)